core/ir/aarch64/instr_create_api.h

```diff
@@ -600,8 +597,9 @@
  * result of a comparison of its two source values if the named input condition
  * is true, or to an immediate value if the input condition is false.
  * \param dc      The void * dcontext used to allocate memory for the #instr_t.
- * \param cond    A 4-bit immediate value created using #OPND_CREATE_COND() specifying
+ * \param cond    A 4-bit immediate value created using #opnd_create_cond() specifying
  *                the condition used for the comparison.
+ *                For example opnd_create_cond(DR_PRED_EQ).
```

Comment:
This may need `#` in front of both: `#opnd_create_cond(#DR_PRED_EQ)` should get a link for each.

Ditto below.

Reply:
Done.